src/app/app.component.html

```diff
@@ -27,4 +27,9 @@
 <ngx-spinner bdOpacity="0.2" bdColor="rgba(51,51,51,0.6)" size="medium" color="#fff" type="ball-clip-rotate-pulse"
   fullScreen="true">
   <p style="color: white"> {{'Loading...' | translate }} </p>
+</ngx-spinner>
+
+<ngx-spinner [name]="'saveconfig'" bdOpacity="0.2" bdColor="rgba(51,51,51,0.6)" size="medium" color="#fff" type="ball-clip-rotate-pulse"
```

Comment:
Why are we not using the spinner that is in the toolkit ? Is that spinner used anywhere ?